### PR TITLE
fix(TDKN-142): add the radio and select widget

### PR DESCRIPTION
**Pull request by wwang-talend**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [x] Tests for the changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [x] The code coverage on new code >75%
- [x] The new code does not introduce new technical issues (sonar / eslint)

**What is the current behavior?** (You can also link to an open issue here)
https://jira.talendforge.org/browse/TDKN-142

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/134/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/134/)

File: daikon/src/test/resources/org/talend/daikon/serialize/jsonschema/FullExampleUiSchema.json [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+{"stringProp":{"ui:trigger":["after","beforePresent","validate"]},"schema":{"ui:widget":"schema","ui:trigger":["after","beforeActive","validate"]},"multipleSelectionProp":{"ui:trigger":["after","beforeActive","validate"]},"showNewForm":{"ui:widget":"button","ui:trigger":["after","beforePresent","validate"]},"commonProp":{"ui:widget":"hidden"},"hiddenTextProp":{"ui:widget":"password","ui:trigger":["after","beforePresent","validate"]},"filepathProp":{"ui:widget":"file","ui:trigger":["after","beforePresent","validate"]},"tableProp":{"ui:widget":"table"},"ui:order":["stringProp","schema","multipleSelectionProp","showNewForm","commonProp","hiddenTextProp","filepathProp","integerProp","longProp","dateProp","tableProp","hideStringPropProp","textareaProp","radioProp","selectProp"],"hideStringPropProp":{"ui:widget":"hidden"},"textareaProp":{"ui:widget":"hidden"},"radioProp":{"ui:widget":"hidden"},"selectProp":{"ui:widget":"hidden"}}
```
Comment (wwang-talend) [inline daikon/src/test/resources/org/talend/daikon/serialize/jsonschema/FullExampleUiSchema.json]: Hi Seb, after your commit, the json format in the test is not like the expected one for dataprep. It miss the part like : 
```
"radioProp":{"ui:widget":"radio","ui:options":{"inline":"true"}},"selectProp":{"ui:widget":"select"}
```

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/139/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/139/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/141/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/141/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/142/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/142/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/143/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/143/)

File: daikon/src/main/java/org/talend/daikon/serialize/jsonschema/UiSchemaConstants.java [hunk 2]
```diff
@@ -12,6 +12,11 @@
 public class UiSchemaConstants {
 
     /**
+     * Build-in tag. Represent the widget type is custom
+     */
+    public static final String TAG_CUSTOM_WIDGET = "ui:field";
```
Comment (wwang-talend) [inline daikon/src/main/java/org/talend/daikon/serialize/jsonschema/UiSchemaConstants.java]: the field is not useful now, it appear here as the git merge action, please move it

**Pull request merged**